<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReoMain</class>
 <widget class="QMainWindow" name="ReoMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reo</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-dictionary">
    <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLineEdit" name="searchEntry">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="searchButton">
          <property name="toolTip">
           <string>Search</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
          <property name="icon">
           <iconset theme="edit-find">
            <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearButton">
          <property name="toolTip">
           <string>Clear</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="icon">
           <iconset theme="edit-clear">
            <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="audioButton">
          <property name="toolTip">
           <string>Audio</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Audio</string>
          </property>
          <property name="icon">
           <iconset theme="audio-speakers-symbolic">
            <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="defView">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionPaste_Search"/>
    <addaction name="actionSearch_Selected"/>
    <addaction name="actionSave_Definition"/>
    <addaction name="actionRandom_Word"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSearch"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionPaste_Search">
   <property name="icon">
    <iconset theme="edit-paste-symbolic">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Paste &amp;&amp; Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
  <action name="actionSearch_Selected">
   <property name="icon">
    <iconset theme="edit-find-symbolic">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Search Selected Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Definition">
   <property name="icon">
    <iconset theme="document-save-symbolic">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Save Definition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRandom_Word">
   <property name="icon">
    <iconset theme="media-playlist-shuffle-symbolic">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Random Word</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about-symbolic">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>About Reo</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="window-close-symbolic">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchEntry</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>defView</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>searchEntry</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
